viz/charts/linkerd-viz/templates/metrics-api.yaml

```diff
@@ -0,0 +1,105 @@
+---
+###
+### Metrics API
+###
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: linkerd-metrics-api
+  namespace: {{.Values.namespace}}
+  labels:
+    {{.Values.extensionAnnotation}}: linkerd-viz
+    component: metrics-api
+  annotations:
+    {{.Values.createdByAnnotation}}: {{default (printf "linkerd/helm %s" .Values.linkerdVersion) .Values.cliVersion}}
+spec:
+  type: ClusterIP
+  selector:
+    {{.Values.extensionAnnotation}}: linkerd-viz
+    component: metrics-api
+  ports:
+  - name: http
+    port: 8085
+    targetPort: 8085
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    {{.Values.createdByAnnotation}}: {{default (printf "linkerd/helm %s" .Values.linkerdVersion) .Values.cliVersion}}
+  labels:
+    {{.Values.extensionAnnotation}}: linkerd-viz
+    app.kubernetes.io/name: metrics-api
+    app.kubernetes.io/part-of: Linkerd
+    app.kubernetes.io/version: {{default .Values.linkerdVersion .Values.controllerImageVersion}}
+    component: metrics-api
+    namespace: {{.Values.namespace}}
```

Comment:
Is the `namespace` label used anywhere? I know a few of the Viz components have this on resources, but I'm not totally sure on the purpose it serves.

This is related to the general cleanup of labels we should do across extensions anyways so not a blocker, but just curious seeing it being added here.

Reply:
Yeah I don't see a purpose for this either, should be removed.